core/src/main/java/io/grpc/internal/DnsNameResolver.java

```diff
@@ -82,9 +84,13 @@
   @GuardedBy("this")
   private Listener listener;
 
-  DnsNameResolver(@Nullable String nsAuthority, String name, Attributes params,
+  DnsNameResolver(
+      @Nullable String nsAuthority,
+      String name,
+      Attributes params,
       Resource<ScheduledExecutorService> timerServiceResource,
-      Resource<ExecutorService> executorResource) {
+      Resource<ExecutorService> executorResource,
+      boolean balancerLookup) {
```

Comment:
No, but it will know how to do SRV record lookups.  SRV is not grpclb specific.  Its more DNS than LB so it belongs there.